src/client.ts

```diff
@@ -1684,6 +1692,21 @@ export class StreamChat<
     });
   }
 
+  /** userMuteStatus - check if a user is muted or not, can be used after setUser() is called
+   *
+   * @param {string} targetID
+   * @returns {boolean}
+   */
+  userMuteStatus(targetID: string) {
+    if (!this.user || !this.wsPromise)
```

Comment:
@vishalnarkhede does this make sense? this function is valid only if you set a user so client.mutes is initialized when the `health.check` is fired.
I think we need the same check for `client._muteStatus()` which handles channel mutes